src/components/activityIndicator.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+
+export default class ActivityIndicator extends Component {
```

Comment:
If your component just render stuff without any other logic, make it a functional component

Reply:
Note that this is just a personal preference, not a change request